### PR TITLE
Fixes #227 : Added contributing practices 

### LINKED ISSUE #227: Contributing Practices missing
- Issue type: Feature request 

### Short description

There must be a document containing contributing practices.

### Steps to reproduce

1. A contributing guideline should be added.

### Usecase
A documentation containing contributing practices.

###

### PR DESCRIPTION
**Pull request by YashKumarVerma**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by tabesin** [comment]
See also https://github.com/fossasia/meilix/pull/226

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* When you make very very minor changes to a PR of yours (like for example fixing a failing travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterwards so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at https://davidwalsh.name/squash-commits-git )
+* When you're submitting a PR for a UI-related issue, it would be really awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker.
+
+**Code Styleguide**
+* Do follow the .editorconfig file regarding maintaining of code style (It's mandatory).
+* For more information regarding .editorconfig file, see [editorconfig](http://editorconfig.org/#download)
+
+**Feature Requests and Bug Reports**
+* When you file a feature request or when you are submitting a bug report to the [issue tracker](https://github.com/fossasia/susper.com/issues), make sure you add steps to reproduce it. Especially if that bug is some weird/rare one.
```
[the remark this answers is not in the document]
Comment (YashKumarVerma) [reply .github/CONTRIBUTING.md]: Oh. Whops. 😶

**Comment by YashKumarVerma** [comment]
So i think i must close this right?

**Comment by tabesin** [comment]
Just add the changes, it could be approved.

**Comment by meets2tarun** [comment]
@YashKumarVerma 
Thanks a lot for your PR. Welcome to the Meilix community !
Please look here 
https://github.com/fossasia/meilix#contributions-best-practices
and make required changes into the PR.

**Pull request merged**